restful.module

```diff
@@ -426,6 +426,20 @@ function restful_menu_access_callback($resource_name, $version = NULL) {
     return;
   }
 
+  // Set the request and method on the handler, so access callbacks have full
+  // access to the request and account.
```

Comment:
Right, then it is for `$handler->access()`. Sorry, I misunderstood your last comment. Carry on.